src/lib/pages/frontsite/Documents/DocumentDetails/DocumentPanel/DocumentPanel.js

```diff
@@ -61,6 +61,10 @@ class DocumentPanel extends Component {
                         listItemAs="h4"
                         limit={invenioConfig.LITERATURE.authors.maxDisplay}
                       />
+                      <Overridable
+                        id="DocumentPanel.StandardNumber"
```

Comment:
I like the idea of having this overridable! I would just maybe have a more generic hook name, e.g. `DocumentPanel.beforeAbstract`.
What about the mobile version?